conftest.py

```diff
@@ -45,7 +45,13 @@ def pytest_report_header(config):
 # `pytest --hypothesis-profile=fuzz ...` argument.
 
 hypothesis.settings.register_profile(
-    "ci", deadline=None, print_blob=True, derandomize=True
+    "ci",
+    deadline=None,
+    print_blob=True,
+    derandomize=True,
+    # disabling HealCheck.differing_executors to allow double test
```

Comment:
Would disabling healthcheck for all CI come back and bite us in the future? The check was added for a reason, right? 🤔 

```suggestion
    # disabling HealhCheck.differing_executors to allow double test
```

Reply:
Yes, but that reason was that you'd get problems if you inherit from a `TestCase` class which already defines a test method decorated with `@given()`, if you got unlucky in what was written to the database.  I think this is very unlikely to affect Astropy.

Reply:
Thanks for your reply. Let's ping @mhvk to be sure... 🤞